pkg/acsengine/testdata/disks-managed/swarm-vmss_expected.json

```diff
@@ -507,6 +507,13 @@
         "description": "SSH public key used for auth to all Linux machines.  Not Required.  If not set, you must provide a password key."
       },
       "type": "string"
+    },
+    "targetEnvironment": {
```

Comment:
Thanks for reviewing it. I am on vacation for Chinese new year and I will send the update once I get back to the office.